Examples/Makefile.in

```diff
@@ -1627,3 +1693,4 @@ d_clean:
 	rm -f *_wrap* *~ .~* $(RUNME) $(RUNME).exe `find . -name \*.d | grep -v $(RUNME).d`
 	rm -f core @EXTRA_CLEAN@
 	rm -f *.@OBJEXT@ *@SO@
+
```

Comment:
Extra blank line added at end of file.

Reply:
This one must be due to some merge weirdness because git-blame in the branch attributes it to e0067ea99 (Merge trunk rev 13025 (swig-2.0.6) into gsoc2008-maciekd, 2012-05-07)... Anyhow, I've removed it now, thanks.